src/components/Synthetics/GmList/GmList.tsx

```diff
@@ -174,117 +182,21 @@ export function GmList({ marketsTokensApyData, marketsTokensIncentiveAprData, sh
-                        useCommas: true,
-                        displayDecimals: 2,
-                      })}
-                      <br />({formatUsd(totalSupplyUsd)})
-                    </ExchangeTd>
-                    <ExchangeTd>
-                      <MintableAmount
-                        mintableInfo={mintableInfo}
-                        market={market}
-                        token={token}
-                        longToken={longToken}
-                        shortToken={shortToken}
-                      />
-                    </ExchangeTd>
-
-                    <ExchangeTd>
-                      <GmTokensBalanceInfo
-                        token={token}
-                        daysConsidered={daysConsidered}
-                        oneLine={false}
-                        earnedRecently={marketEarnings?.recent}
-                        earnedTotal={marketEarnings?.total}
-                      />
-                    </ExchangeTd>
-
-                    <ExchangeTd>
-                      <AprInfo apy={apy} incentiveApr={incentiveApr} tokenAddress={token.address} />
-                    </ExchangeTd>
-
-                    <ExchangeTd className="w-[350px]">
-                      <div className="flex flex-wrap gap-10">
-                        <Button
-                          className="flex-grow"
-                          variant="secondary"
-                          to={`/pools/?market=${market.marketTokenAddress}&operation=buy&scroll=${
-                            shouldScrollToTop ? "1" : "0"
-                          }`}
-                        >
-                          <Trans>Buy</Trans>
-                        </Button>
-                        <Button
-                          className="flex-grow"
-                          variant="secondary"
-                          to={`/pools/?market=${market.marketTokenAddress}&operation=sell&scroll=${
-                            shouldScrollToTop ? "1" : "0"
-                          }`}
-                        >
-                          <Trans>Sell</Trans>
-                        </Button>
-                      </div>
-                    </ExchangeTd>
-                  </ExchangeTr>
-                );
-              })}
+              currentData.map((token) => (
+                <GmListItem
+                  key={token.address}
+                  marketsInfoData={marketsInfoData}
+                  tokensData={tokensData}
+                  token={token}
+                  marketsTokensApyData={marketsTokensApyData}
+                  marketsTokensIncentiveAprData={marketsTokensIncentiveAprData}
+                  userEarnings={userEarnings}
+                  showDebugValues={showDebugValues}
+                  daysConsidered={daysConsidered}
+                  shouldScrollToTop={shouldScrollToTop}
+                  isShiftAvailable={shiftAvailableMarketAddressSet.has(token.address)}
```

Comment:
props tokensData, marketsInfoData, userEarnings, showDebugValues, daysConsidered can be omitted for simplicity, wdyt?

Reply:
moved them inside list item